internal/cli/root/atlas/builder.go

```diff
@@ -189,3 +203,38 @@ func formattedVersion() string {
 		runtime.GOARCH,
 		runtime.Compiler)
 }
+
+func (c *Notifier) shouldCheck() bool {
+	return !config.SkipUpdateCheck() && cli.IsTerminal(c.writer)
+}
+
+func (c *Notifier) notifyIfApplicable(isHb bool) error {
+	release, err := c.finder.Find()
+	if err != nil || release == nil {
+		return err
+	}
+
+	if isHb && !isAtLeast24HoursPast(release.PublishedAt) {
```

Comment:
let's add a comment here
```suggestion
	// hombrew is an external dependency we give them 24h to have the cli available there
	if isHb && !isAtLeast24HoursPast(release.PublishedAt) {
```
so the context of the 24h is not lost